lib/aws-record/record/attributes.rb

```diff
@@ -20,8 +20,37 @@ def self.included(sub_class)
         model_attributes = ModelAttributes.new(self)
         sub_class.instance_variable_set("@attributes", model_attributes)
         sub_class.instance_variable_set("@keys", KeyAttributes.new(model_attributes))
+
+        if sub_class.superclass.include?(Aws::Record)
```

Comment:
Thanks for the tip! I updated to have these blocks into private methods ✨
I didn't see the need to make a specific note that code is for inheritance since I felt the method names was descriptive. Let me know what you think!